src/components/02-cards/event/_event.scss

```diff
@@ -0,0 +1,52 @@
+.sfgov-event-card {
+  @include rubik;
+  background: $green-3;
+  border: 0;
+  border-radius: 8px;
```

Comment:
I have a `$radius` variable for this in `00-design-tokens/_variables`

Reply:
missed that, thanks